app/store/initial_state.js

```diff
@@ -13,7 +13,10 @@ const state = {
             appState: false,
             credentials: {},
             config: {},
-            dataRetentionPolicy: {},
+            dataRetention: {
+                policies: {},
+                lastCleanUpAt: null,
+            },
```

Comment:
What is the impact of this change with servers that do not have granular data retention support or even older apps that had it enabled?

Reply:
When getting the data retention policies, we are checking for the supported server version and in case server doesn't support this, we only get global policy.